vmaas/common/config.py

```diff
@@ -38,6 +39,7 @@ def __init__(self):
 
         self.oval_unfixed_eval_enabled = strtobool(os.getenv("OVAL_UNFIXED_EVAL_ENABLED", "FALSE"))
 
+        self.repo_name_prefixes = os.getenv("REPO_NAME_PREFIXES", "") # Comma separated prefixes
```

Comment:
Thank you for input! I agree with you, but on the other hand reading env vars in one place is easier in maintenance for users and that is what I have seen in most codebases before. Are you strongly opinionated about this one?

Reply:
There are already some env vars in webapp.py. I think it is better to read env vars in one place esp. for smaller projects. But we have bigger project with more components so env vars reading is a bit spread. Also some common parameters are provided by clowder operator but we still need some way how to read it from env vars for local and legacy run method. I think that's the reason the Config class was created. Otherwise it would be easier to read it as a global var as you don't need to create any Config object. But for specific parameters (not provided by clowder) it's still better to use global var approach. Also it's better to parse it directly into the data structure you use in the code (array in this case) then parse it every time again :wink: .